fenix/app/src/main/java/org/mozilla/fenix/translations/preferences/automatic/PrefAutomaticTranslationOptions.kt

```diff
@@ -0,0 +1,120 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations.preferences.automatic
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.semantics.clearAndSetSemantics
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.button.RadioButton
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Firefox Automatic Translation Options fragment compose view.
+ *
+ * @param selectedOption Selected option that will come from Gecko.
```

Comment:
nit: I keep using the name "Gecko" too, but we should try to avoid using platform specific language!

```suggestion
 * @param selectedOption Selected option that will come from the translations engine.
```

Reply:
Done